ports/nrf/README.md

```diff
@@ -106,6 +107,7 @@ wt51822_s4at         | s110                    | Peripheral             | Manual
 pca10040             | s132                    | Peripheral and Central | [Segger](#segger-targets)
 feather52            | s132                    | Peripheral and Central | Manual, SWDIO and SWCLK solder points on the bottom side of the board
 arduino_primo        | s132                    | Peripheral and Central | [PyOCD](#pyocdopenocd-targets)
+[wrl13990](ports/wrl13990)| s132                    | Peripheral and Central | [Segger](#segger-targets)
```

Comment:
A minor syntactic sugar detail, but could you expand the "Target board (BOARD)" with a few spaces, and the also add some blank spaces on the other targets so that the PIPE characters get aligned?